.github/workflows/main.yml

```diff
@@ -31,7 +46,7 @@ jobs:
           --health-interval 10s
           --health-timeout 5s
           --health-retries 5
-
+          
```

Comment:
Remove blank space from the line

Reply:
why? I haven''t added it , it's there on the develop branch too

Reply:
> why? I haven''t added it , it's there on the develop branch too

The diff shows you have.

Reply:
no idea why it shows 👀